### DISCUSSION TITLE
Double entries from Adobe Acrobat

### DISCUSSION THREAD

**Discussion by NVHT** [moudey/Shell#83] [General]
![image](https://user-images.githubusercontent.com/8341213/197504591-9359ee8b-8fee-42a4-9337-28ed21383948.png)

Is there a way to hide the entry with the tick?

**Comment by moudey** [comment]
```
static
{
    item(find='convert to adobe pdf'
         where=this.checked 
         vis=remove)
}
```

**Comment by NVHT** [comment]
![image](https://user-images.githubusercontent.com/8341213/198010868-26afb18d-ca76-408b-b69f-b412de8fd076.png)
Also how can I replicate this from the default taskbar menu?

**Comment by moudey** [comment]
What is the Windows version number you are using?

**Comment by NVHT** [comment]
> What is the Windows version number you are using?

Win 11, Shell 1.7

**Comment by moudey** [comment]
Looks like you got your old taskbar back. If so what program did you use.
Please try beta version 1.8

**Comment by NVHT** [comment]
> Looks like you got your old taskbar back. If so what program did you use. Please try beta version 1.8

No, that's not the screenshot I took from my computer. It is just reference on how to make the taskbar menu to look like that.
